src/main/java/core/org/akaza/openclinica/domain/datamap/Study.java

```diff
@@ -776,7 +778,8 @@ public void setStudies(List<Study> studies) {
      * }
      */
 
-    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.LAZY, mappedBy = "study")
+    @OneToMany(targetEntity=StudyParameterValue.class,cascade = CascadeType.ALL, fetch = FetchType.EAGER, mappedBy = "study")
+    @Fetch(value = FetchMode.SUBSELECT)
```

Comment:
instead of making this eager, use StudyConfigService#setStudyParameterValueToStudyManually

Reply:
done

Reply:
revert this change completely